GameData/B9_Aerospace_HX/README.md

```diff
@@ -0,0 +1,499 @@
+# Installation
```

Comment:
The readme and changelog are copied as part of the build so no need to add them here.

Reply:
ok? ...These files simply appeared as part of the fork and I simply edited the changelog. Should I PM you the changelog contents then?

Reply:
I was maintaining a single changelog for the time being.  It might eventually be better to split them out, but I don't think just copying the files is sufficient